httpbin/helpers.py

```diff
@@ -209,6 +217,10 @@ def status_code(code):
                 'x-more-info': 'http://vimeo.com/22053820'
             }
         ),
+        406: dict(data=json.dumps({
```

Comment:
2 last nits:

1. Can you update the message to be closer to a sentence like in my example
1. Can you also add the appropriate headers, e.g.,

   ```py
   dict(data=json.dumps({
             # ...
         }),
         headers={'Content-Type': 'application/json'})
   ```